src/Data/Reader/FilterHandler/QueryBuilderFilterHandler.php

```diff
@@ -2,9 +2,9 @@
 
 declare(strict_types=1);
 
-namespace Yiisoft\Yii\Cycle\Data\Reader\Processor;
+namespace Yiisoft\Yii\Cycle\Data\Reader\FilterHandler;
 
-interface QueryBuilderProcessor
+interface QueryBuilderFilterHandler
```

Comment:
Move it to namespace  `Yiisoft\Yii\Cycle\Data\Reader` for consistency with https://github.com/yiisoft/data/blob/master/src/Reader/Iterable/IterableFilterHandlerInterface.php